test/cli/test/public-api.test.ts

```diff
@@ -94,6 +94,7 @@ it.each([
     expect(files[0].tasks[index + 1].name).toBe(`custom ${name}`)
     expect(files[0].tasks[index + 1].location).toEqual({
       line: 18,
+      // FIXME: rolldown (possibly) gives incorrect location
```

Comment:
I'm suspecting that this might the bug on esbuild transform, which required the following workaround

https://github.com/vitest-dev/vitest/blob/41cc1d8a48bf866fa3bd4eed1fabc0dc5cf7383a/packages/runner/src/suite.ts#L851-L857

For rolldown vite / oxc, we can remove the extra `+ 1` and the result should matches as plain js stack.

Here is a comparison of source map:
(btw, I made this quick dirty stackblitz app in https://github.com/hi-ogawa/reproductions/tree/main/oxc-sourcemap-debug)

![image](https://github.com/user-attachments/assets/d9f34ecb-6aff-4534-b43d-8daa3aac17d3)